frontend/src/components/features/trips/itinerary/ItineraryMapCanvas.tsx

```
@@ -0,0 +1,400 @@
+        container: containerRef.current,
+        style: buildStyle(mode),
+        center: points[0] ? [points[0].longitude, points[0].latitude] : [0, 20],
+        zoom: points.length > 0 ? 5 : 1.2,
+        attributionControl: true,
+      })
+      mapInstance = map as unknown as typeof mapInstance
+
+      function addOverlays() {
+        if (!mapInstance) return
+
+        markers.forEach((marker) => marker.remove())
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)